### ISSUE TITLE
There are still few endpoints which are not following v5 standards

### ISSUE THREAD

**Issue by vikasrohit** [topcoder-platform/tc-message-service#119]
<img width="1413" alt="Screen Shot 2021-04-09 at 3 51 22 PM" src="https://user-images.githubusercontent.com/2417632/114166510-70dc1300-994b-11eb-98f5-7480716a66fa.png">

fyi @maxceem

**Comment by maxceem** [comment]
Yeah, this was on our list a long time ago. Let me know if you would like to cover it in some shape up or cooldown period.

**Comment by vikasrohit** [comment]
Yep, I would like to wrap it in our next cool down. Do you have the link of the pending tasks handy? I am not able to locate it.

**Comment by maxceem** [comment]
I only had it in my personal todo list backlog, but I couldn't find it logged in git anyhweree.

**Comment by vikasrohit** [comment]
Ok. I think we did had that list. Anyway, lets tackle whatever you have in your list in next cool down period.

**Comment by maxceem** [comment]
@vikasrohit at the moment inside Message Service we are using the old format of Kafka events:
```js
    TOPIC_CREATED: 'connect.notification.project.topic.created',
    TOPIC_UPDATED: 'connect.notification.project.topic.updated',
    TOPIC_DELETED: 'connect.notification.project.topic.deleted',
    POST_CREATED: 'connect.notification.project.post.created',
    POST_UPDATED: 'connect.notification.project.post.edited',
    POST_DELETED: 'connect.notification.project.post.deleted',
```

while as per V5 it should be like:
```
messages.action.created
messages.action.updated
messages.action.deleted
```

with `reference` field inside which would say what kind of object it is like:
- `topic`
- `post`
- `post-attachment`
- `post-user-stats`

Should we update the Kafka events to the new format?

**Comment by vikasrohit** [comment]
Yes, lets do it. We don't want to do the multiple QA regression rounds.

**Comment by maxceem** [comment]
#### V5 API Endpoints

Update all the endpoints to follow [Topcoder V5 API Standards](https://platform.topcoder-dev.com/model/standards/api-standards):
- Use HTTP verbs "GET", "POST", "PATCH", "DELETE" instead of adding a verb to the path, see [screenshot](https://monosnap.com/file/WgsWoNAtLan0louAqVnx9XKpiwLRn6).
- Update the shape of responses so it does not wrap responses into object "result.content" and so on, see [screenshot](https://monosnap.com/file/2TsjBf3VV7tGldaeYLKRafXySQkrA8).
- For lists of objects return pagination information inside the header like [this](https://github.com/topcoder-platform/taas-apis/blob/cf5201fb636eed7b2753b21603fab261fffdcd76/src/common/helper.js#L588-L620).
- Update Swagger and Postman.
